.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,91 @@
+name: 🐛 Bug Report
+description: Report a bug encountered while using Cluster API
+labels: kind/bug
```

Comment:
Q: sometimes uses files issues as a kind/bug but then we switch to something else, like e.g kind/support or kind/documentation etc. Will this be possible also when using forms?

Reply:
Not when we use the labels field here. If we add it as one of the text fields it should be preserved 1:1

Reply:
Good question, I have not thought about it. I will try that use case out and come back, as @sbueringer text field should be the way forward

Reply:
I have moved `labels: kind/bug` to the text field only in the bug_report issue template so it is preserved 1:1 as @sbueringer suggested